tests/testthat/test-vector3.R

```diff
@@ -157,6 +157,81 @@ test_that("Vector3 distances are correctly computed", {
   )
 })
 
+test_that("Vector3 distances are correctly computed with euclidean norm", {
+  froms <- vector3(
+    x = c(1, -4, 4, 3),
+    y = c(0, -3, 5, 4),
+    z = c(-2, 5, -7, 0)
+  )
+  
+  expect_equal(
+    distance_between(froms, to = vector3(0, 2, -3), "euclidean"),
+    c(sqrt(6), sqrt(105), sqrt(41), sqrt(22))
+  )
+})
+
+test_that("Vector3 distances are correctly computed with manhattan norm", {
+  froms <- vector3(
+    x = c(1, -4, 4, 3),
+    y = c(0, -3, 5, 4),
+    z = c(-2, 5, -7, 0)
+  )
```

Comment:
I'd merge these statements into one `test_that` call with multiple `expect_equal` calls - I just looked it up, and the library author says this about tests: "A test groups together multiple expectations to test one function, or tightly related functionality across multiple functions. A test is created with the test_that function." (https://vita.had.co.nz/papers/testthat.pdf)

Reply:
Merged into one test! Still expected 8 failed tests after consolidating into one `test_that` call